public/js/monitor.js

```diff
@@ -306,6 +306,67 @@ function addBentoObserver(){
     });
   });
 
+  document.querySelectorAll(".relay-sign-up-btn").forEach(btn => {
+    btn.addEventListener("click", async(e) => {
+      const relayEndpoint = new URL("/relay-waitlist", document.body.dataset.serverUrl);
+      const signUpCallout = document.querySelector(".relay-sign-up");
+
+      signUpCallout.classList.add("sending-email");
+      try {
+        await fetch(relayEndpoint, {
+          headers: {
+            "Content-Type": "application/json; charset=utf-8",
+          },
+          mode: "same-origin",
+          method: "POST",
+          body: JSON.stringify({"emailToAdd": "add-user-email"}),
+        })
+        .then(async(data) => {
```

Comment:
Should we use `then` when we're using `await`?

Reply:
It works but what I just pushed up should be easier to read.